service/static/js-src/muk.funder.js

```diff
@@ -4004,36 +4004,27 @@ $.extend(true, edges, {
+                            results += "</label></div></div>"
+                        } else {
+                            results += '<div class="form ' + resultClass + '"><div class="form-fields__item-checkbox ' + valClass + '" data-key="' + edges.escapeHtml(val.term) + '"><label><input type="checkbox" /> ' + edges.escapeHtml(val.display);
                             if (this.showCount) {
-                                filterFrag += " (" + def.count + ")"
+                                results += ' <span class="' + countClass + '">(' + val.count + ")</span>"
                             }
-                            filterFrag += '&nbsp;<a href="#" class="' + filterRemoveClass + '" data-key="' + edges.escapeHtml(def.term) + '">';
-                            filterFrag += '<i class="glyphicon glyphicon-black glyphicon-remove"></i></a>';
-                            filterFrag += "</strong></a></div>"
+                            results += "</label></div></div>"
                         }
                     }
                 }
+
                 var header = this.headerLayout({
                     toggleId: toggleId
                 });
-                var frag = '<div class="' + facetClass + '">                        <div class="' + headerClass + '"><div class="row">                             <div class="col-md-12">                                ' + header + '                            </div>                        </div></div>                        <div class="' + bodyClass + '">                            <div class="row" style="display:none" id="' + resultsId + '">                                <div class="col-md-12">                                    {{SELECTED}}                                </div>                                <div class="col-md-12"><div class="' + selectionsClass + '">                                    {{RESULTS}}                                </div>                            </div>                        </div>                        </div></div>';
-                frag = frag.replace(/{{RESULTS}}/g, results).replace(/{{SELECTED}}/g, filterFrag);
+                var frag = '<div class="' + facetClass + '">                        <div class="' + headerClass + '"><div class="row">                             <div class="col-md-12">                                ' + header + '                            </div>                        </div></div>                        <div class="' + bodyClass + '">                            <div class="row" style="display:none" id="' + resultsId + '">  <div class="col-md-12"><div class="' + selectionsClass + '">                                    {{RESULTS}}                                </div>                            </div>                        </div>                        </div></div>';
+                frag = frag.replace(/{{RESULTS}}/g, results);
```

Comment:
(1) No idea, but I will remove every space that is not needed.
(2) Before we had multiple replaces. I will make the change as replace is no longer needed.